cli/src/pcluster/templates/imagebuilder_stack.py

```diff
@@ -92,8 +92,8 @@ def __init__(self, scope: core.Construct, construct_id: str, imagebuild: ImageBu
         # InfrastructureConfiguration
         imagebuilder.CfnInfrastructureConfiguration(
             self,
-            id="PClusterImageInfrastructureConfiguration",
-            name="-".join(["PCluster-Image-Infrastructure-Configuration", resources_prefix]),
+            id="ParallelClusterImageInfrastructureConfiguration",
+            name="-".join(["ParallelCluster-Image-Infrastructure-Configuration", resources_prefix]),
```

Comment:
we can be consistent in the naming of all the resources and use the convention
`name=id-resources_prefix`, i.e.
`name="-".join(["ParallelClusterInfrastructureConfiguration", resources_prefix]),`

Reply:
done